### ISSUE TITLE
[tool_crash] ExistingDartDevelopmentServiceException

### ISSUE THREAD

**Issue by Prince2347X** [flutter/flutter#135552]
## Command
```
flutter run -d chrome --web-port 5000 --web-renderer html
```

## Steps to Reproduce
1. ...
2. ...
3. ...

## Logs
ExistingDartDevelopmentServiceException
```
#0      DartDevelopmentServiceImpl._startDDSServer (package:dds/src/dds_impl.dart:227:7)
<asynchronous suspension>
#1      DartDevelopmentServiceImpl.startService (package:dds/src/dds_impl.dart:119:7)
<asynchronous suspension>
#2      DartDevelopmentService.startDartDevelopmentService (package:dds/dds.dart:97:5)
<asynchronous suspension>
#3      DebugService.startDartDevelopmentService (package:dwds/src/services/debug_service.dart:165:12)
<asynchronous suspension>
#4      DevHandler._createAppDebugServices (package:dwds/src/handlers/dev_handler.dart:514:7)
<asynchronous suspension>
#5      DevHandler.loadAppServices (package:dwds/src/handlers/dev_handler.dart:239:21)
<asynchronous suspension>
#6      Dwds.debugConnection (package:dwds/dart_web_debug_service.dart:59:30)
<asynchronous suspension>
#7      WebDevFS.connect.<anonymous closure> (package:flutter_tools/src/isolated/devfs_web.dart:715:15)
<asynchronous suspension>
```
```
[32m[✓][39m Flutter (Channel stable, 3.13.3, on Microsoft Windows [Version 10.0.19045.3448], locale en-IN)
    [32m•[39m Flutter version 3.13.3 on channel stable at C:\Users\princ\flutter
    [32m•[39m Upstream repository https://github.com/flutter/flutter.git
    [32m•[39m Framework revision 2524052335 (3 weeks ago), 2023-09-06 14:32:31 -0700
    [32m•[39m Engine revision b8d35810e9
    [32m•[39m Dart version 3.1.1
    [32m•[39m DevTools version 2.25.0

[32m[✓][39m Windows Version (Installed version of Windows is version 10 or higher)

[32m[✓][39m Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    [32m•[39m Android SDK at C:\Users\princ\AppData\Local\Android\sdk
    [32m•[39m Platform android-34, build-tools 34.0.0
    [32m•[39m Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-10027231)
    [32m•[39m All Android licenses accepted.

[32m[✓][39m Chrome - develop for the web
    [32m•[39m Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[31m[✗][39m Visual Studio - develop Windows apps
    [31m✗[39m Visual Studio not installed; this is necessary to develop Windows apps.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components

[32m[✓][39m Android Studio (version 2022.3)
    [32m•[39m Android Studio at C:\Program Files\Android\Android Studio
    [32m•[39m Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    [32m•[39m Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    [32m•[39m android-studio-dir = C:\Program Files\Android\Android Studio
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-10027231)

[32m[✓][39m VS Code (version 1.82.2)
    [32m•[39m VS Code at C:\Users\princ\AppData\Local\Programs\Microsoft VS Code
    [32m•[39m Flutter extension version 3.72.0

[32m[✓][39m Connected device (3 available)
    [32m•[39m Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19045.3448]
    [32m•[39m Chrome (web)      • chrome  • web-javascript • Google Chrome 116.0.5845.188
    [32m•[39m Edge (web)        • edge    • web-javascript • Microsoft Edge 117.0.2045.41

[32m[✓][39m Network resources
    [32m•[39m All expected network resources are available.

[33m![39m Doctor found issues in 1 category.

```

## Flutter Application Metadata
**Type**: app
**Version**: 1.0.0+1
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: stable
**Creation framework version**: 4f9d92fbbdf072a70a70d2179a9f87392b94104c
### Plugins
audio_session-0.1.16
emoji_picker_flutter-1.6.1
flutter_web_auth_2-2.2.1
fluttertoast-8.2.2
google_sign_in-6.1.4
google_sign_in_android-6.1.18
google_sign_in_ios-5.6.2
google_sign_in_web-0.12.0+2
just_audio-0.9.35
just_audio_web-0.4.8
package_info_plus-4.1.0
path_provider-2.0.15
path_provider_android-2.0.27
path_provider_foundation-2.2.4
path_provider_linux-2.1.11
path_provider_windows-2.1.7
shared_preferences-2.2.0
shared_preferences_android-2.2.0
shared_preferences_foundation-2.3.2
shared_preferences_linux-2.3.0
shared_preferences_web-2.2.0
shared_preferences_windows-2.3.0
sqflite-2.3.0
url_launcher-6.1.12
url_launcher_android-6.0.37
url_launcher_ios-6.1.4
url_launcher_linux-3.0.5
url_launcher_macos-3.0.6
url_launcher_web-2.0.18
url_launcher_windows-3.0.7
video_player-2.7.1
video_player_android-2.4.10
video_player_avfoundation-2.4.10
video_player_web-2.0.17
wakelock_plus-1.1.1
webview_flutter-4.2.4
webview_flutter_android-3.9.3
webview_flutter_wkwebview-3.7.3
window_to_front-0.0.3

**Comment by danagbemava-nc** [comment]
Hi @Prince2347X, thanks for reporting this. 

How often do you get this crash?

**Comment by Prince2347X** [comment]
> Hi @Prince2347X, thanks for reporting this.
> 
> How often do you get this crash?

I got this error only once and I followed the link to report it (creating this issue).

**Comment by danagbemava-nc** [comment]
Thanks, labeling for further insight from the team. 

This seems similar to other dds errors in terms of the stacktrace but the error message is different.

**Comment by bkonyi** [comment]
This issue is likely no longer relevant since DDS is launched using `dart development-service` now.
